config/TLS-example/make_credentials

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
```

Comment:
Why is this script located in this folder? Also, it should be named as `make_credentials.sh`